src/Extension/BaseElementExtension.php

```diff
@@ -161,58 +164,14 @@ public function updateCMSFields(FieldList $fields)
     }
 
     /**
-     * Ensure that if there are elements that are virtualised from this element
-     * that we move the original element to replace one of the virtual elements
+     * Client specific requirement https://mbiessd.atlassian.net/browse/MWP-390
```

Comment:
?

Reply:
🤦 why did I created this pr on module!
I will close this pr. Sorry about that.